src/NuGet.Clients/NuGet.PackageManagement.UI/GlobalSuppressions.cs

```diff
@@ -132,3 +132,4 @@
 [assembly: SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.PackageManagement.UI.PackagesConfigToPackageReferenceMigrator.DoUpgradeAsync(NuGet.PackageManagement.UI.INuGetUIContext,NuGet.PackageManagement.UI.INuGetUI,NuGet.VisualStudio.Internal.Contracts.IProjectContextInfo,System.Collections.Generic.IEnumerable{NuGet.PackageManagement.UI.NuGetProjectUpgradeDependencyItem},System.Collections.Generic.IEnumerable{NuGet.Packaging.Core.PackageIdentity},System.IProgress{NuGet.PackageManagement.UI.ProgressDialogData},System.Threading.CancellationToken)~System.Threading.Tasks.Task{System.String}")]
 [assembly: SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.PackageManagement.UI.PackageSolutionDetailControlModel.UpdateInstalledVersionsAsync(System.Threading.CancellationToken)~System.Threading.Tasks.Task")]
 [assembly: SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.PackageManagement.UI.UIActionEngine.UpgradeNuGetProjectAsync(NuGet.PackageManagement.UI.INuGetUI,NuGet.VisualStudio.Internal.Contracts.IProjectContextInfo)~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("Build", "CA1501:'VulnerabilitiesControl' has an object hierarchy '9' levels deep within the defining module. If possible, eliminate base classes within the hierarchy to decrease its hierarchy level below '6': 'UserControl, ContentControl, Control, FrameworkElement, UIElement, Visual, DependencyObject, DispatcherObject, Object'", Justification = "<Pending>", Scope = "type", Target = "~T:NuGet.PackageManagement.UI.VulnerabilitiesControl")]
```

Comment:
Normally, I would add a comment on anew CA suppresion:

```suggestion
[assembly: SuppressMessage("Build", "CA1501:'VulnerabilitiesControl' has an object hierarchy '9' levels deep within the defining module. If possible, eliminate base classes within the hierarchy to decrease its hierarchy level below '6': 'UserControl, ContentControl, Control, FrameworkElement, UIElement, Visual, DependencyObject, DispatcherObject, Object'", Justification = "Default WPF class hierarchy", Scope = "type", Target = "~T:NuGet.PackageManagement.UI.VulnerabilitiesControl")]
```

Reply:
Updated locally. Changes will be reflected with next commit / push.